_episodes/02-lists.md

```diff
@@ -290,33 +272,6 @@ odds: [1, 3, 5, 7]
 This is different from how variables worked in lesson 1, and more similar to how a spreadsheet
 works.
 
-> ## Turn a String Into a List
->
-> Use a for-loop to convert the string "hello" into a list of letters:
->
-> ~~~
-> ["h", "e", "l", "l", "o"]
-> ~~~
-> {: .language-python}
->
-> Hint: You can create an empty list like this:
->
-> ~~~
-> my_list = []
-> ~~~
-> {: .language-python}
->
-> > ## Solution
-> > ~~~
-> > my_list = []
-> > for char in "hello":
-> >     my_list.append(char)
-> > print(my_list)
-> > ~~~
-> > {: .language-python}
-> {: .solution}
-{: .challenge}
-
```

Comment:
Could you please move this exercise to `_extras/extra_exercises.md` instead of removing it completely?